docs/maintainers.md

```diff
@@ -191,20 +191,12 @@ This will publish the Lambda layer to the AWS China (Beijing) Region.
 9. **Merge docs PR**: Once the `Layer Deployment (Partition)` workflow for the production China partition is complete,
 merge the PR from step 4 to update the documentation with the new version.
 10. **Update SSM Parameters (Beta)**: Run the `SSM Parameters` workflow with the `main` branch, targeting the `beta`
-deployment environment, and using the package version from npm (i.e., `2.20.0`) and Lambda layer version from step 4.
+deployment environment, and using the package version from npm (i.e., `2.20.0`).
 This will update the SSM parameters with the new version.
-11. **Verify SSM Parameters (Beta)**: Use the AWS CLI to verify that the SSM parameters were updated correctly. Run
-the following command: `aws ssm get-parameter --name=/aws/service/powertools/beta/typescript/generic/all/latest`
-and `aws ssm get-parameter --name=/aws/service/powertools/beta/typescript/generic/all/<version>` to verify that the
-SSM parameters were updated correctly.
-12. **Update SSM Parameters (Prod)**: Run the `SSM Parameters` workflow with the `main` branch, targeting the `prod`
-deployment environment, and using the package version from npm (i.e., `2.20.0`) and Lambda layer version from step 4.
+11. **Update SSM Parameters (Prod)**: Run the `SSM Parameters` workflow with the `main` branch, targeting the `prod`
+deployment environment, and using the package version from npm (i.e., `2.20.0`).
```

Comment:
Not for this PR but I'm wondering if we can automate the part where we get the version from NPM: `npm view @aws-lambda-powertools/logger version` rather than have the user input it.

Reply:
We can also `cat packages/logger/package.json | jq .version`